docs/tailwind.config.js

```diff
@@ -21,32 +29,29 @@ module.exports = {
       cursor: {
         inherit: 'inherit',
       },
-      border: {
-        3: '3px',
+      fontFamily: {
+        sans: ['IBM Plex Sans', ...defaultTheme.fontFamily.sans],
       },
```

Comment:
This makes me think that at one point we could consider having an MUI theme to Tailwind CSS theme converter like in https://reshaped.so/content/docs/getting-started/integrations/tailwind.